scalableminds/webknossos
# remove user agent tracking, fixes #2095

### Mailable description of changes:

### Steps to test:
- drop `userAgentTracking` collection, load wK, check that collection has not been recreated

### Issues:
- fixes #2095 

------
